src/modules/aws_iam@0.0.1/entity/role.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Column,
+  Entity,
+  PrimaryColumn,
+} from 'typeorm';
+
+import { cloudId, } from '../../../services/cloud-id'
+
+// TODO complete schema
+@Entity()
+export class Role {
+  @Column({
+    nullable: true,
+  })
+  @cloudId
+  arn?: string;
+
+  // Guaranteed unique in AWS
+  // Maximum 128 characters. Use alphanumeric and '+=,.@-_' characters.
+  @PrimaryColumn()
+  name: string;
```

Comment:
Hmm I'm not sure about this, I think this should not be decorated and add the following property to keep consistency between entities: 
```
@PrimaryGeneratedColumn()
  id?: number;
```

Reply:
why? the role name is guaranteed unique in aws, so it simplifies references to the table and reduces aws calls since all their methods just take the role name